aedes.js

```diff
@@ -270,6 +271,10 @@ Aedes.prototype.unregisterClient = function (client) {
   this.connectedClients--
   delete this.clients[client.id]
   this.emit('clientDisconnect', client)
+  this.publish({
+    topic: '$SYS/' + this.id + '/new/clientDisconnect',
+    payload: new Buffer(client.id, 'utf8')
+  }, noop)
```

Comment:
I think this should be named differently. Would you mind checking what's the situation in Mosquitto and Mosca?